src/components/Modal/BaseModal.js

```diff
@@ -15,11 +15,15 @@ const propTypes = {
 
     /** The ref to the modal container */
     forwardedRef: PropTypes.func,
+
+    /** Whether the modal should go under the system statusbar */
+    statusBarTranslucent: PropTypes.bool,
```

Comment:
@Tushu17 this should be moved to `modalPropTypes.js` because external modals will be setting this prop.

Reply:
Good catch!

Reply:
Made the changes.